README.adoc

```diff
@@ -17,6 +17,10 @@
 :source-highlighter: prettify
 = Creating a MicroProfile application
 
+[.hidden]
+NOTE: This guide is replaced by https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service
```

Comment:
How about: The _Creating a MicroProfile application_ guide has been archived and is replaced by the https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service] guide and the https://openliberty.io/guides/cdi-intro.html[Injecting dependencies into microservices] guide.